scrnaseq.Rmd

```diff
@@ -902,16 +920,17 @@ p
 # Default assay is set automatically
 if (param$integrate_samples[["method"]]=="single") {
   sc = sc[[1]]
-  message("Your dataset contains 1 sample only. No merging/intregating required.")
+  message("Your dataset contains 1 sample only. No merging/integrating required.")
 }
 ```
 
 ```{r multi_dataset_integration_preparation, eval=(param$integrate_samples[["method"]]!="single"), include=(param$integrate_samples[["method"]]!="single"), results="asis"}
 if (param$integrate_samples[["method"]]!="single") {
   
-  # When merging, feature meta-data is removed by Seurat entirely; save separately for each assay and add again afterwards
-  assay_names = unique(purrr::flatten_chr(purrr::map(list_names(sc), function(n) { Seurat::Assays(sc[[n]]) } )))
-  
+  # When merging, feature meta-data is removed by Seurat entirely; save separately for each assay except for SCT and add again afterwards
+  # Note: not sure whether this is still needed - discuss
```

Comment:
Why is it not needed for `SCT`?

Reply:
The metadata for SCT features is less meaningful - it is basically only the variable genes info. But see for yourself please